app/services/CollectionService.scala

```diff
@@ -97,14 +97,15 @@ object CollectionService {
       config: ConfigJson,
       containerService: ContainerService
   ): Option[StoriesVisibleByStage] = {
+    val cConfigJson = config.collections.get(collectionId).get
```

Comment:
This will throw an exception of the collection is not found, please refactor to handle the failure case and avoid the `.get`